### ISSUE TITLE
New buzzwords Is It Time to Replace Social Security Numbers?

### ISSUE THREAD

**Issue by postmillenial** [postmillenial/reading-list#769]
Is It Time to Replace Social Security Numbers?<br>
http://ift.tt/2gaaUBC<br><img width="604" height="450" src="http://ift.tt/2icT2uo" class="webfeedsFeaturedVisual wp-post-image wp-stateless-item" style="display: block; margin: auto; margin-bottom: 20px;" data-image-size="large" data-stateless-media-bucket="cdn.thenewstack.io" data-stateless-media-name="media/2017/10/7ab929ec-c964032f-socialsecurity.cardpunching.ssa_.jpg"><p>It may be the ultimate data problem. How do we allow people to identify themselves digitally&nbsp;in a way that&rsquo;s secure yet simple? After a <a href="http://ift.tt/2ylUNfp" class="ext-link" rel="external nofollow">massive data breach at Equifax</a> in which 143 million social security numbers were accessed, America has begun searching for a better solution.</p>
<p>In fact, security technologist <a href="http://ift.tt/xgwZum" class="ext-link" rel="external nofollow">Bruce Schneier</a> thinks social security numbers were just filling a void as the country transitioned to a digital society. &ldquo;They appeared at an age when we didn&rsquo;t have other numbers,&rdquo; Schneier said in an interview <a href="http://ift.tt/2kDcuB7" class="ext-link" rel="external nofollow">with <em>Insurance Journal</em></a>. &ldquo;Think of this as part of our aging infrastructure&hellip; Sooner or later we as a society need to fix our aging infrastructure.&rdquo;</p>
<h2>How We Got Here</h2>
<p>Originally the sole purpose of the nine-digit number set was to identify Americans for their retirement payments from the social security system &mdash; as well as their pre-retirement contributions. Since nearly every working American <em>has</em> a social security number, it&rsquo;s since morphed into a catch-all identification number. A tax reform law in 1990 even required parents to list social security numbers <a href="http://ift.tt/2h9KDok" class="ext-link" rel="external nofollow">for any child over the age of one</a> (if they wanted a tax deduction for them), and the Social Security Administration now advises parents to apply for the number <a href="http://ift.tt/1ShJ2et" class="ext-link" rel="external nofollow">when their children are born</a>.</p>
<p>Other countries have entirely different systems&nbsp;&mdash; and in some cases, they&rsquo;re more sophisticated. India uses a unique number issued only after the intended recipient has been fingerprinted and had their iris scanned&nbsp;&mdash; and it&rsquo;s used by <a href="http://ift.tt/KisnBF" class="ext-link" rel="external nofollow">almost 1.2 billion people</a>. China uses a similar system, named Hukou,&nbsp;<a href="http://ift.tt/2idsU2C" class="ext-link" rel="external nofollow">based on residence permits</a>. Estonia implemented a system of national ID cards that are cryptographically-secure using a distributed blockchain ledger that enables <a href="http://ift.tt/2gHcZ8V" class="ext-link" rel="external nofollow">additional applications in healthcare, travel papers, and even for identifying voters</a>. &ldquo;It&rsquo;s safe to say that well over a third of the world uses government-issued identifiers,&rdquo; noted <a href="http://ift.tt/2gdxTfS" class="ext-link" rel="external nofollow">a recent essay</a> in the <em>Harvard Business Review</em>&nbsp;&mdash; and most of them go beyond a simple nine-digit number.</p>
<p>So Thursday a U.S. Congressman introduced new legislation that would require credit bureaus to <a href="http://ift.tt/2g3ITvM" class="ext-link" rel="external nofollow">phase out the use of Social Security numbers by 2020</a>.</p>
<p>And Rob Joyce, the White House&rsquo;s cybersecurity coordinator, says the White House already asked federal departments and agencies to <a href="http://ift.tt/2hK0DjT" class="ext-link" rel="external nofollow">research ways to replace social security numbers</a> with &ldquo;modern cryptographic identifiers&rdquo; like public and private keys. Joyce is also expressing his strong opinion that the familiar nine-digit number &ldquo;has outlived its usefulness.&rdquo;</p>
<h2>Looking for a Change</h2>
<p>Phasing out social security numbers is not a new idea, <a href="http://ift.tt/2ylq3Kp" class="ext-link" rel="external nofollow">notes <em>Wired</em></a>. The Obama administration also investigated security digital identity options through a program called the <a href="http://ift.tt/2ibpnBF" class="ext-link" rel="external nofollow">National Strategies for Trusted Identities in Cyberspace</a>. &ldquo;It&rsquo;s going to be a while until this problem is solved,&rdquo; says <a href="http://ift.tt/2wTkJvb" class="ext-link" rel="external nofollow">Paul Grassi</a>, a senior standards and technology advisor at the <a href="http://www.nist.gov" class="ext-link" rel="external nofollow">National Institute of Standards and Technology</a>, &ldquo;but we built our latest guidelines under the assumption that your data is out there whether you like it or not.&rdquo;</p>
<p>Richard Smith &mdash; the recently-retired CEO of Equifax &mdash; agrees. Earlier this month he told a Congressional committee it&rsquo;s time for America to &ldquo;<a href="http://ift.tt/2hK0DjT" class="ext-link" rel="external nofollow">think beyond</a>&rdquo; the idea of social security numbers being private and secure.</p>
<p>There&rsquo;s a lot of interest in the idea from the financial community. Analysts at Cowen Inc. believe this approach could save the credit bureau industry since &ldquo;this reduces the risk of business-model-busting legislation such as a requirement that consumers opt-in to a credit bureau collecting their data.&rdquo; When Equifax&rsquo;s former CEO appeared before Congress, Illinois Representative <a href="http://ift.tt/1hkK8Sl" class="ext-link" rel="external nofollow">Jan Schakowsky</a> pointed out that &ldquo;<a href="http://ift.tt/2ylUNfp" class="ext-link" rel="external nofollow">I never opted in</a>. I never said it was okay to have all my information, and now I want out&hellip; Can I do that?&rdquo;</p>
<p>&ldquo;That requires a much broader discussion,&rdquo; Smith replied, &ldquo;around the role of the credit reporting agencies.&rdquo;</p>
<p><a href="http://ift.tt/PCritB" class="ext-link" rel="external nofollow">Steven M. Bellovin</a>, a professor of computer science at Columbia University, says he&rsquo;s also heard arguments for the elimination of credit bureaus, but &ldquo;the correct answer is stricter controls on what information can be kept and on how it can be used,&rdquo; <a href="http://ift.tt/2fNbl4S" class="ext-link" rel="external nofollow">he told Motherboard</a>.</p>
<p>But everyone agrees there&rsquo;s a lot of money at stake. In August Equifax&rsquo;s then CEO Richard Smith told students at the University of Georgia that credit bureaus get their data for free from banks &mdash; then sells it back to the banks after performing a computerized analysis, giving the company <a href="http://ift.tt/2xGE2IW" class="ext-link" rel="external nofollow">a gross margin of about 90 percent</a>.</p>
<p></p>
<h2>What Next?</h2>
<p>There&rsquo;s just one problem. &ldquo;It&rsquo;s really hard to do better,&rdquo; argues professor Bellovin. &ldquo;Basically, running any sort of national identity scheme is hard, and it&rsquo;s not clear that the replacement would have fewer problems than what we have now.&rdquo; Calling the nine-digit numbers &ldquo;a low-grade secret,&rdquo; he argues that they&rsquo;re used as a database key to uniquely identify records and link them properly to the same individual&rsquo;s data in other databases. The depressing truth: &ldquo;if you need linkage and you need memorability to recover from lost credentials, any replacement for the social security number is going to have most or all of the same problems.&rdquo;</p>
<p>Schneier sees another issue. While more secure systems are possible, &ldquo;magic math costs money.&rdquo;</p>
<p>Maybe one solution involves making it even more expensive to have an <em>insecure</em> system. Bellovin notes that some financial firms send a paper mail to a physical address to confirm changes &mdash; but that this isn&rsquo;t implemented consistently, which &ldquo;suggests that the real solution is regulatory: make credit providers liable for the full damages, including ongoing inconvenience, suffered by victims of identity theft.&rdquo;</p>
<p>One Republican congressman from Texas has even suggested &ldquo;some kind of fine-per-account-hacked that&rsquo;s large enough that even a company that&rsquo;s worth $13 billion would rather protect their data and probably not collect as much data,&rdquo; <a href="http://ift.tt/2xehTzz" class="ext-link" rel="external nofollow">Bloomberg reported</a>.</p>
<p>Another suggested fix: having a <em>mix</em> of identifiers&nbsp;&mdash; possibly interoperable, but definitely more decentralized, reducing the impact of any single breach. This has the added benefit of reducing what the <a href="https://www.aclu.org/" class="ext-link" rel="external nofollow">American Civil Liberties Union</a>&nbsp;senior policy analyst <a href="http://ift.tt/1b9Q00r" class="ext-link" rel="external nofollow">Jay Stanley</a> calls the &ldquo;eagle&rsquo;s eye view of every activity.&rdquo;</p>
<p>&ldquo;There is a clear need for individuals to be identified and authenticated and there are ways to do it that still preserve privacy,&rdquo; he <a href="http://ift.tt/2ylq3Kp" class="ext-link" rel="external nofollow">told Wired Friday</a>. &ldquo;People use the Social Security number because they don&rsquo;t have anything else. It&rsquo;s ridiculous.&rdquo;</p>
<p><em>Insurance Journal</em> also spoke to <a href="http://ift.tt/2gG9dge" class="ext-link" rel="external nofollow">Bob Stasio</a>, a fellow at the <a href="http://ift.tt/RFBjD3" class="ext-link" rel="external nofollow">Truman National Security Project</a>&nbsp;who discussed the possibility of using a public key, which would then authorize the sending of messages to be decrypted by the corresponding <em>private</em> key. One big advantage? It&rsquo;s easier to replace the private keys in the event of a data breach than it is to replace a social security number.</p>
<p>Another possibility is biology-based &ldquo;biometric&rdquo; identification systems. For example, Apple&rsquo;s newest iPhone X uses 3D sensing cameras to <a href="http://ift.tt/2i94bfP" class="ext-link" rel="external nofollow">recognize the face of its owner</a>, replacing the Touch ID system on older iPhones which used fingerprints. They&rsquo;re not foolproof. Researchers were able to the new iPhone by creating a 3D rendering of a face <a href="http://ift.tt/2bsqnw8" class="ext-link" rel="external nofollow">using photos from Facebook</a>, and another team successfully created a set of fake &ldquo;MasterPrints&rdquo; combining common features from many fingerprints which fooled fingerprint identification systems <a href="http://ift.tt/2nA18ip" class="ext-link" rel="external nofollow">up to 65 percent of the time</a>.</p>
<p>But security lawyer <a href="http://ift.tt/2icT6dC" class="ext-link" rel="external nofollow">Jerri-Lynn Scofield</a> isn&rsquo;t a fan of biometric solutions for another reason. &ldquo;What we&rsquo;re being asked to do is to turn over our biometric information, and then trust those to whom we do so to safeguard that data,&rdquo; she wrote recently <a href="http://ift.tt/2g9YB92" class="ext-link" rel="external nofollow">on an economics blog</a>. Given the current status of database security, corporate and governmental accountability, etc.: How do you think that is going to play out?&rdquo;</p>
<p>Beyond security issues, there&rsquo;s an issue with the data collection itself. &ldquo;It should be apparent that part of the problem is that the centralization of so much valuable information is what draws hackers. And the more we centralize, the more precious the prize will be&hellip; Inviting the biometric ID fairy drop by and replace the existing Social Security number is not the solution.&rdquo;</p>
<p>And ultimately the Equifax breach happened in part because the company <a href="http://ift.tt/2xC8IxH" class="local-link">failed to patch a known security issue</a>, suggesting some of our problems aren&rsquo;t technical so much as they are procedural. During a hearing with Equifax&rsquo;s retired CEO, Oregon Congressman <a href="http://ift.tt/2izr0Ht" class="ext-link" rel="external nofollow">Greg Walden</a> was even more skeptical that <em>any</em> legislative fix could compensate for egregious human errors.</p>
<p>As he told <a href="http://ift.tt/2fN5z6T" class="ext-link" rel="external nofollow">a local television station</a>, &ldquo;I don&rsquo;t think we can pass a law that can fix stupid.&rdquo;</p>
<hr><h1>WebReduce</h1>
<ul>
<li>Hackers latest target for surreptitiously mining bitcoins? <a href="http://ift.tt/2g1Nqm1" class="ext-link" rel="external nofollow">Two AWS customers</a>.</li>
<li>Would you trust Google&rsquo;s AI to <a href="http://ift.tt/2icT85e" class="ext-link" rel="external nofollow">take candid photos without asking?</a>
</li>
<li>Why book publisher Tim O&rsquo;Reilly isn&rsquo;t pessimistic about biased algorithms&nbsp;&mdash; and why he&rsquo;s <a href="http://ift.tt/2xgjOJ6" class="ext-link" rel="external nofollow">optimistic about the future</a>.</li>
<li>More than one-third of marriages now start online. New data-based evidence that <a href="http://ift.tt/2ye1lf6" class="ext-link" rel="external nofollow">online dating is changing society</a>.</li>
<li>It&rsquo;s the 25th anniversary of <a href="http://ift.tt/2xUcySH" class="ext-link" rel="external nofollow">IBM&rsquo;s ThinkPad laptop</a><br><i>And the 60th anniversary of <a href="http://ift.tt/2w5dhfn" class="ext-link" rel="external nofollow">Fairchild Semiconductor</a>.</i>
</li>
<li>Fans share their fond farewells <a href="http://ift.tt/2wCsipp" class="ext-link" rel="external nofollow">to AOL Instant Messenger</a>&nbsp;&mdash; including <a href="http://ift.tt/2y2d1SM" class="ext-link" rel="external nofollow">Mark Zuckerberg</a>.</li>
<li>Parody &ldquo;SubGenius&rdquo; church tries to crowdfund <a href="http://ift.tt/2wFaEkP" class="ext-link" rel="external nofollow">an alien-contacting beacon</a>.</li>
</ul><p class="attribution">Feature image:&nbsp;Pre-computer cardpunch operators at America&rsquo;s Social Security Administration, from Wikipedia, public domain.</p>
<p>The post <a rel="nofollow" href="http://ift.tt/2gaaUBC">Is It Time to Replace Social Security Numbers?</a> appeared first on <a rel="nofollow" href="https://thenewstack.io">The New Stack</a>.</p>
<br><br>
via Blog &ndash; The New Stack https://thenewstack.io<br>
October 15, 2017 at 09:00AM
